<ui version="4.0" >
 <class>UiPreferenceBase</class>
 <widget class="QWidget" name="UiPreferenceBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>760</width>
    <height>435</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>760</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Preference</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../resource/dms.qrc" >:/picture/16/images/16x16/preference_16.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="2" row="0" column="0" >
    <widget class="QListWidget" name="listWidgetPreferences" >
     <property name="minimumSize" >
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="showDropIndicator" stdset="0" >
      <bool>false</bool>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="movement" >
      <enum>QListView::Static</enum>
     </property>
     <property name="flow" >
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0" >
      <bool>true</bool>
     </property>
     <property name="resizeMode" >
      <enum>QListView::Fixed</enum>
     </property>
     <property name="gridSize" >
      <size>
       <width>128</width>
       <height>100</height>
      </size>
     </property>
     <property name="viewMode" >
      <enum>QListView::IconMode</enum>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="currentRow" >
      <number>0</number>
     </property>
     <item>
      <property name="text" >
       <string>Database</string>
      </property>
      <property name="icon" >
       <iconset resource="../../resource/dms.qrc" >:/picture/32/images/32x32/add_32.png</iconset>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>File associations</string>
      </property>
      <property name="icon" >
       <iconset resource="../../resource/dms.qrc" >:/picture/32/images/32x32/documents_32.png</iconset>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Plugins</string>
      </property>
      <property name="icon" >
       <iconset resource="../../resource/dms.qrc" >:/picture/32/images/32x32/applications_32.png</iconset>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Skins</string>
      </property>
      <property name="icon" >
       <iconset resource="../../resource/dms.qrc" >:/picture/32/images/32x32/skin_32.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageDatabase" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="labelDatabase" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape" >
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text" >
          <string>Database</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageFileAssociations" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="labelFileAssociations" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape" >
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text" >
          <string>File associations</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="groupBoxDocuments" >
         <property name="title" >
          <string>Documents</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="labelApplication" >
              <property name="minimumSize" >
               <size>
                <width>110</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>110</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>Application:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditProgram" >
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSelectApplication" >
              <property name="minimumSize" >
               <size>
                <width>28</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>28</width>
                <height>26</height>
               </size>
              </property>
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="labelFileExtansion" >
              <property name="minimumSize" >
               <size>
                <width>110</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>110</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>Fileextansion:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit" >
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" >
           <widget class="QTreeWidget" name="treeWidgetApplicationPref" >
            <property name="rootIsDecorated" >
             <bool>false</bool>
            </property>
            <column>
             <property name="text" >
              <string>Programm</string>
             </property>
            </column>
            <column>
             <property name="text" >
              <string>Fileextansion</string>
             </property>
            </column>
            <item>
             <property name="text" >
              <string>/usr/bin/oowriter</string>
             </property>
             <property name="text" >
              <string>odt;ott;sxw;stw;sdw;doc</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>/usr/bin/oocalc</string>
             </property>
             <property name="text" >
              <string>ods;ots;sxc;stc;sdc;fods;xls;xlt;csv</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>/opt/kde3/bin/ark</string>
             </property>
             <property name="text" >
              <string>7z;zip;tar;tar.gz;tar.bz2;gz;tgz;bzip;bzip2</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="btnAdd" >
              <property name="minimumSize" >
               <size>
                <width>110</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>110</width>
                <height>26</height>
               </size>
              </property>
              <property name="text" >
               <string>A&amp;dd</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnUpdate" >
              <property name="minimumSize" >
               <size>
                <width>110</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>110</width>
                <height>26</height>
               </size>
              </property>
              <property name="text" >
               <string>&amp;Update</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnRemove" >
              <property name="minimumSize" >
               <size>
                <width>110</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>110</width>
                <height>26</height>
               </size>
              </property>
              <property name="text" >
               <string>&amp;Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>91</width>
                <height>26</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pagePlugins" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="labelPlugins" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape" >
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text" >
          <string>Plugins</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="labelPlugins_2" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape" >
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text" >
          <string>Skins</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="btnApply" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>A&amp;pply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidgetPreferences</tabstop>
  <tabstop>btnSelectApplication</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>treeWidgetApplicationPref</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnApply</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>lineEditProgram</tabstop>
 </tabstops>
 <resources>
  <include location="../../resource/dms.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>listWidgetPreferences</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>138</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel" >
     <x>172</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
